data/phi0134/phi004/phi0134.phi004.perseus-eng2.xml

```diff
@@ -2,9 +2,9 @@
    <teiHeader>
       <fileDesc>
          <titleStmt>
-            <title>Phormio, or The Scheming Parasite</title>
-            <author>P. Terentius Afer (Terence)</author>
-            <editor role="transl">Henry Thomas Riley</editor>
+            <title>The Scheming Parasite</title>
```

Comment:
@AlisonBabeu 
I have a question about the English title modifications as I think that the print does indeed read "Phormio, or The Scheming Parasite" so I wouldn't make this change. This probably applies elsewhere?

Reply:
Ok, I wasn't sure. I will change it back so it reads exactly as in the printed volume for the header.

Reply:
Yes, I think those titles for all of that set of translations are as the print reads. I know they look like something we might have added, but I don't think that's true. The work files should be consistent with that, too.
(I only looked at Phormio).

Reply:
I added in page level links and you are right that is how they show up in the printed volume. I will change it in the header and the cts work level